assets/js/modules/optimize/common/setup-information.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * SetupInformation component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import Link from '../../../components/link';
+
+export default function SetupInformation() {
```

Comment:
Since this component is used in both setup and settings forms, it's a bit odd to name it with "Setup".

I needed a similar component for the GTM store that I called "FormInstructions" which seems like it would fit well here too.

Reply:
Done